### PR TITLE
Update API end to end URLs

### PR DESCRIPTION
**Pull request by elyezer**
Also set maxDiff to None in order to get full diff when a failure happens.

### CODE REVIEW

**Comment by elyezer** [comment]
I am not sure why those URLs were removed, I just make sure a 0 failure run for these tests since they are going to be integrated to the release process.

I ask help from automation folks to check if these are valid removals or not.

File: tests/foreman/endtoend/test_api_endtoend.py [hunk 1]
```diff
@@ -217,11 +216,6 @@
         u'/docker/api/v2/containers/:id/logs',
         u'/docker/api/v2/containers/:id/power',
     ),
-    u'content_reports':(
-        u'/katello/api/content_reports/status_trend',
-        u'/katello/api/content_reports/system_status',
-        u'/katello/api/content_reports/system_trend',
-    ),
```
Comment (sghai) [inline tests/foreman/endtoend/test_api_endtoend.py]: Content-reports URL's are related to gutterball feature, which is going to be completely removed from Sat6.2. So I think that removal is valid.

File: tests/foreman/endtoend/test_api_endtoend.py [hunk 2]
```diff
@@ -61,7 +61,6 @@
         u'/katello/api/activation_keys/:id/product_content',
         u'/katello/api/activation_keys/:id/releases',
         u'/katello/api/activation_keys/:id/remove_subscriptions',
-        u'/katello/api/activation_keys/:id/subscriptions',
```
Comment (oshtaier) [inline tests/foreman/endtoend/test_api_endtoend.py]: Hmm, in that case, I am not sure how we can receive list of subscriptions that we just added to activation key

**Comment by ehelms** [comment]
All tests passing for me after applying this patch.

File: tests/foreman/endtoend/test_api_endtoend.py (hunk 2 above)
Comment (oshtaier) [inline tests/foreman/endtoend/test_api_endtoend.py]: Seems as regression. Anyway, if that change is intended, we need to make corresponding change to nailgun too

Comment (elyezer) [inline tests/foreman/endtoend/test_api_endtoend.py]: There is an endpoint on the subscriptions that can provide that information.

**Comment by oshtaier** [comment]
ACK pending related nailgun PR

**Comment by elyezer** [comment]
@oshtaier this don't have a relation with NailGun. If NailGun is using the removed end point then it should be updated, this end to test just helps us identify when upcoming changes to the API paths are coming or not. The test did it's job and showed that we have changes on the API, the next steps are not related to the update of this test.

**Comment by elyezer** [comment]
Also if you look the `subscriptions` endpoints you will see `/katello/api/activation_keys/:activation_key_id/subscriptions` which is the same and the removed one, in other words both pats were duplicated. The only different is that on subscriptions the AK ID is called `:activation_key_id` and on the activation key it is just `:id` but they expect the same value.

This looks like as just proper information being returned.

**Comment by elyezer** [comment]
I've checked the Robottelo tests and found the following:

```
$ git grep "\.subscriptions("
tests/foreman/cli/test_activationkey.py:        ak_subs_info = ActivationKey.subscriptions({
tests/foreman/cli/test_activationkey.py:        ak_subs_info = ActivationKey.subscriptions({
tests/foreman/cli/test_activationkey.py:        result = ActivationKey.subscriptions({
```

That robottelo is not using the subscriptions method of the ActivationKey entity.

**Comment by omaciel** [comment]
ACK. Talked about this with @elyezer and this seems to be harmless. Merging.

**Pull request merged**